code/modules/holiday/holidays.dm

```diff
@@ -314,6 +316,30 @@
 	begin_day = 14
 	begin_month = DECEMBER
 
+/datum/holiday/thanksgiving
+	name = "Thanksgiving"
+	begin_week = 4
+	begin_month = NOVEMBER
+	begin_weekday = THURSDAY
+
+/datum/holiday/columbus
+	name = "Columbus Day"
+	begin_week = 2
+	begin_month = OCTOBER
+	begin_weekday = MONDAY
+
+/datum/holiday/mother
+	name = "Mothers' Day"
+	begin_week = 2
+	begin_month = MAY
```

Comment:
Most countries celebrate Mothers' Day on the 2nd Sunday in May, but in the UK it is celebrated based on the month of Lent, so that can't be calculated based on this, sorry.